packages/sdks/web-component/src/lib/descope-wc/sdkScripts/forter.ts

```diff
@@ -0,0 +1,29 @@
+const forterScript = `
```

Comment:
maybe add a comment to where this script is taken from

Reply:
this code was provided to us by Forter team, I added a comment noting that 👍